### PR TITLE
Add default arguments to `AnimatedPhysicalModel`

### PR DESCRIPTION
**Pull request by nate-thegrate**
Currently, `PhysicalModel` has [default arguments](https://github.com/flutter/flutter/blob/2e806700b9287c4b4f9fc49fbdfbd81089fc65dd/packages/flutter/lib/src/widgets/basic.dart#L1093) for `shape` and `elevation`, but `AnimatedPhysicalModel` [does not](https://github.com/flutter/flutter/blob/2e806700b9287c4b4f9fc49fbdfbd81089fc65dd/packages/flutter/lib/src/widgets/implicit_animations.dart#L1998).

This pull request makes both classes consistent.

### CODE REVIEW

**Comment by Hixie** [comment]
We should still test this, ideally (checking that there is a default and that it's the one we expect).

File: packages/flutter/lib/src/widgets/implicit_animations.dart [hunk 1]
```diff
@@ -1995,10 +1995,10 @@ class AnimatedPhysicalModel extends ImplicitlyAnimatedWidget {
   const AnimatedPhysicalModel({
     super.key,
     required this.child,
-    required this.shape,
+    this.shape = BoxShape.rectangle,
     this.clipBehavior = Clip.none,
-    this.borderRadius = BorderRadius.zero,
-    required this.elevation,
+    this.borderRadius,
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/implicit_animations.dart:2000]: Why remove the explicit default value for borderRadius?

Comment (nate-thegrate) [reply packages/flutter/lib/src/widgets/implicit_animations.dart:2000]: The non-animated class has a nullable border radius, so this change makes them consistent.

**Review by goderbauer — APPROVED** [review]
LGTM

**Comment by nate-thegrate** [comment]
Guess who's able to add labels now 😎😎😎

**Comment by goderbauer** [comment]
> Guess who's able to add labels now 😎😎😎

🥳 Welcome to [flutter-hackers](https://github.com/orgs/flutter/teams/flutter-hackers)!

**Pull request merged**